appveyor.yml

```diff
@@ -1,32 +0,0 @@
-version: 1.0.{build}
```

Comment:
We don't need it anymore?

Reply:
No, we don't need it.  It is just needed when we need a configuration different from the defaults configured on the server.